prov/efa/Makefile.include

```diff
@@ -73,10 +73,30 @@ _efa_headers = \
 	prov/efa/src/rxr/rxr_atomic.h \
 	prov/efa/src/rxr/rdm_proto_v4.h
 
+
+if ENABLE_EFA_UNIT_TEST
+check_PROGRAMS += prov/efa/test/efa_unit_test
+TESTS += prov/efa/test/efa_unit_test
+nodist_prov_efa_test_efa_unit_test_SOURCES = \
+	prov/efa/test/efa_unit_tests.h \
+	prov/efa/test/efa_unit_tests.c \
+	prov/efa/test/rdma_core_mocks.c
+
+prov_efa_test_efa_unit_test_LDADD = $(efatest_LIBS) $(linkback)
+prov_efa_test_efa_unit_test_CPPFLAGS = -I$(top_srcdir)/include -pthread $(efatest_CPPFLAGS) -I$(top_srcdir)/prov/efa/src -I$(top_srcdir)/prov/efa/src/rxr -I$(top_srcdir)/prov/efa/test
```

Comment:
It seems that we can use `$efa_CPPFLAGS $cmocka_CPPFLAGS`? we do not really need `efatest_CPPFLAGS`.

Reply:
removed efatest_CPPFLAGS